blockchain/My-Wallet-V3-Android
# Feature/ppv2 btc

## Objective

To support Lunu ppv2 payment protocol.

## Description

A lot of changes have been made. I've added superclasses to implement additional ppv2 providers.

## How to Test

Got to https://shop.rc.lunu.io/.
Choose googs. Got to basket. Click checkout. Fill out form fields with any test data.
Check agree and the press pay.
Choose Trust Wallet.
Choose bitcoin.
Click Pay.
Scan QR and pay.
Note: your payment will not go through for now. It it just for PR review.

## Screenshot/Design assessment

*If the change is UI related, paste a screenshot here and include an assessment of design accuracy if applicable.*

## Merge Checklist

- [ ] You have added `contentDescription` tags to new Views
